Source/Shaders/PolylineShadowVolumeFS.glsl

```diff
@@ -0,0 +1,81 @@
+#ifdef GL_EXT_frag_depth
+#extension GL_EXT_frag_depth : enable
+#endif
+
+varying vec4 v_startPlaneNormalEC_and_halfWidth;
+varying vec4 v_endPlaneNormalEC_and_batchId;
+varying vec4 v_rightPlaneEC; // Technically can compute distance for this here
+varying vec4 v_ecEnd_and_ecStart_X;
```

Comment:
"ec" here and below are eye coordinate, right?  Make it a suffix like it is elsewhere, e.g., "endEC", not "ecEnd"